homeassistant/components/geniushub/switch.py

```diff
@@ -35,6 +71,23 @@ def device_class(self):
         """Return the class of this device, from component DEVICE_CLASSES."""
         return DEVICE_CLASS_OUTLET
 
+    async def _refresh(self, payload: Optional[dict] = None) -> None:
+        """Process any signals."""
+        if payload is None:
```

Comment:
WOn't you need a `super()._refresh()` here?

Reply:
I'm not sure. It works as it in. I will add `super()._refresh()` and test.

Reply:
No `super()._refresh()` available at this point in the code.

I will see about moving the function.

Reply:
I think this is better as it stands.